indico/modules/categories/serialize.py

```diff
@@ -82,7 +83,7 @@ def serialize_categories_ical(category_ids, user, event_filter=True, event_filte
                     if event.venue_name and event.room_name
                     else (event.venue_name or event.room_name))
         cal_event = ical.Event()
-        cal_event.add('uid', u'indico-event-{}@cern.ch'.format(event.id))
+        cal_event.add('uid', u'indico-event-{}@{}'.format(event.id, gethostname()))
```

Comment:
i don't think using the hostname of the machine is a good idea. it'll break in multi-worker environment and also discloses the hostnames of the workers to users. why not use the host of the base url? `url_parse(config.BASE_URI).host`

Reply:
> also discloses the hostnames of the workers to users

good point